### PR TITLE
docs: Administration guide: caching images for faster workspace start

### PR DESCRIPTION
**Pull request by ibuziuk**
## What does this pull request change
Administration guide: caching images for faster workspace start

image

## What issues does this pull request fix or reference
Part of the https://github.com/eclipse/che/issues/21283

## Specify the version of the product this pull request applies to

for `master` branch
cherry-picking to the `main` branch is expected to be handled by the docs team

## Pull request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [x] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
- [x] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [x] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/administration-guide/pages/defining-the-list-of-images-to-pull.adoc [hunk 1]
```diff
@@ -7,39 +7,16 @@
...
-* The `jq` tool is available. See link:https://stedolan.github.io/jq/[jq homepage].
-* The `yq` tool is available. See link:https://pypi.org/project/yq/[yq homepage].
-
 .Procedure
 
-. Gather a list of relevant container images for the {platforms-name} platform:
-+
-include::example$snip_{project-context}-getting-the-list-of-relevant-images.adoc[]
-
+. Gather a list of relevant container images for prepulling by navigating to the `pass:c,a,q[{prod-url}]/plugin-registry/v3/external_images.txt`
```
Comment (Eva-Lotte) [inline modules/administration-guide/pages/defining-the-list-of-images-to-pull.adoc]:
```suggestion
. Gather a list of relevant container images for pre-pulling by navigating to the `pass:c,a,q[{prod-url}]/plugin-registry/v3/external_images.txt` file.
```

File: modules/administration-guide/pages/defining-the-list-of-images-to-pull.adoc [hunk 2]
```diff
@@ -7,39 +7,16 @@
...
 
-. Determine images from the list for pre-pulling. 
-+
-For faster workspace startup times, consider pre-pulling the workspace-related images:
-+
-include::example$snip_{project-context}-list-of-images-for-faster-workspace-start.adoc[]
+. Determine images from the list for pre-pulling. For faster workspace startup times, consider pre-pulling workspace related images such as `che-theia`, `che-machine-exec`, `che-theia-endpoint-runtime-binary` as well as plug-in sidecar images.
```
Comment (Eva-Lotte) [inline modules/administration-guide/pages/defining-the-list-of-images-to-pull.adoc]:
```suggestion
. Determine images from the list for pre-pulling. For faster workspace startup times, consider pre-pulling workspace related images such as `che-theia`, `che-machine-exec`, `che-theia-endpoint-runtime-binary`, and plug-in sidecar images.
```

File: modules/administration-guide/pages/defining-the-list-of-images-to-pull.adoc (hunk 1 above)
Comment (ibuziuk) [reply modules/administration-guide/pages/defining-the-list-of-images-to-pull.adoc]: well, technically this is a URL, not a file
@Eva-Lotte should we use URL instead?

Comment (Eva-Lotte) [reply modules/administration-guide/pages/defining-the-list-of-images-to-pull.adoc]: Ah, I see! In that case yes, using URL will make sense.

Comment (ibuziuk) [reply modules/administration-guide/pages/defining-the-list-of-images-to-pull.adoc]: fixed

**Review by Eva-Lotte — APPROVED** [review]

**Pull request merged**